docs/src/development/development.md

```diff
@@ -0,0 +1,95 @@
+# Developer Guide
+
+## Install prerequisites
+
+1. Install [go][go]
+    - Get the latest patch version for go v1.17.
+2. Install [KIND][kind]
+    - `GO111MODULE="on" go get sigs.k8s.io/kind@v<stableVersion>`.
+3. Install [Kustomize][kustomize]
+    - `brew install kustomize` on macOS.
+    - `choco install kustomize` on Windows.
+    - [install instructions][kustomizelinux] on Linux
+4. Install [envsubst][envsubst]
+   - `go get github.com/a8m/envsubst/cmd/envsubst`
+5. Install make.
+   - `xcode-select --install` on macOS.
+   - `sudo apt-get install build-essential` on Ubuntu Linux 
+
+## Fork and get the source
+
+Fork the [cluster-api-provider-oci repo](https://github.com/oracle/cluster-api-provider-oci):
+
+```bash
+cd "$(go env GOPATH)"/src
+mkdir sigs.k8s.io
+cd sigs.k8s.io/
+git clone git@github.com:<GITHUB USERNAME>/cluster-api-provider-oci.git
+cd cluster-api-provider-oci
+git remote add upstream git@github.com:orale/cluster-api-provider-oci.git
+git fetch upstream
+```
+
+## Running local management cluster for development
+
+The simplest way to test the code is to run it from local.
+If you have `capoci-controller-manager` running in your management cluster,
+please scale down the deployment:
+
+```bash
+kubectl scale deployment/capoci-controller-manager  --replicas=0 -n cluster-api-provider-oci-system
+```
+
+Create and modify the auth-config.yaml file:
```

Comment:
Please wrap `auth-config.yaml` in back ticks as it refers to a literal file name.